platform/libs.jsvg/manifest.mf

```diff
@@ -1,5 +1,6 @@
 Manifest-Version: 1.0
-OpenIDE-Module: org.netbeans.libs.jsvg
-OpenIDE-Module-Implementation-Version: 1
+OpenIDE-Module: org.netbeans.libs.jsvg/2
+OpenIDE-Module-Implementation-Version: 1.7.1
+OpenIDE-Module-Specification-Version: 2.0
```

Comment:
made it 2.0 instead of 2.0.0 since all other spec manifests seem to use only two components.

updated projext.xml of `platform/openide.util.ui.svg` to make it compile